packages/pds/src/config/env.ts

```diff
@@ -74,6 +74,9 @@ export const readEnv = (): ServerEnvironment => {
 
     // secrets
     jwtSecret: envStr('PDS_JWT_SECRET'),
+    jwtSigningKeyK256PrivateKeyHex: envStr(
+      'PDS_JWT_SIGNING_KEY_K256_PRIVATE_KEY_HEX',
```

Comment:
heh or we could do what bryan was pushing for & use multicodecs for our private keys to avoid stuff like this

could jam some multikey <> der utilities in @atproto/crypto

Reply:
I have a smattering of general thoughts on this, not sure how they work out together yet 🥴 😆 

 - For the purposes of configuration I think it would be pretty nice to support PEMs since so much tooling works well with it, and it's familiar to people.  Ideally we could accept something that openssl can spit out.  It's possible that these env vars could become file paths to keys.  Right now here's how keys are generated with common tools—not the end of the world, but feels like it could be better: https://github.com/bluesky-social/pds/blob/01dbd27728754dc765316e5e4a768380238bddc0/installer.sh#L16.
 - For the @atproto/crypto interfaces, I think it would be great in general to be able to convert to node's `KeyObject` interface and/or the webby `CryptoKey` / `CryptoKeyPair` interfaces (I lean towards the latter).  Seems very desirable to me for interop purposes!
 - Just an observation: we'll also want to configure public keys on their own, e.g. the jwt verify key in #1742.  Maybe we'll want a standard way to represent those in @atproto/crypto?

The multicodecs direction seems great for a ton of use-cases, but I'm not sure whether it's ideal for these app configurations.  I might need to learn a little more about how it would work, though.